src/test/javascript/lib/helpers/getters.js

```diff
@@ -0,0 +1,7 @@
+// we have several places we are pulling the same exact keys out of response objects
```

Comment:
I feel like this extra abstraction isn't necessary currently. Calling `getProp('statusCode', response)` is just as readable as `getStatusCode(response)`. Same with `getUris` and `getResponseBody`.

Putting `getStatusCode` and `getResponseBody` in an HTTP lib and `getUris` in a Hub lib might make sense in the future.